src/components/GoogleAuthentication/GoogleAuthentication.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from "react";
+import { gapi, loadAuth2 } from "gapi-script";
+
+//import UserCard from "./UserCard";
+import "./GoogleAuthentication.css";
+
+class GoogleAuthentication extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
We can add everything related to user state to the src/Login/Login component. We can name it "googleUser" in the Login component.